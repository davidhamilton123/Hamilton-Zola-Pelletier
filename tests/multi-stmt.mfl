val x := 3 + 5 * 2
val y := 4 + x
x > y