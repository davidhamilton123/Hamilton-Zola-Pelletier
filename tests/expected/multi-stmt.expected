Prog(
  Val[x](
    BinOp[ADD](
      Token(INT(3))
      BinOp[MULT](
        Token(INT(5))
        Token(INT(2))
      )
    )
  )
  Val[y](
    BinOp[ADD](
      Token(INT(4))
      Token(ID(x))
    )
  )
  RelOp[GT](
    Token(ID(x))
    Token(ID(y))
  )
)
